src/utils/pythonUtils.ts

```diff
@@ -35,6 +36,11 @@ export function inferPythonArgs(projectType: PythonProjectType, ports: number[])
                 '--nothreading',
                 '--noreload'
```

Comment:
Do you guys know why when debugging the `noreload` option on both `Flask` and `Django` is activated? :thinking:

Reply:
This disables the hot-reloading feature. By default, our Docker container build is set up to copy in the files; since the files are copied in instead of being bind mounted, hot-reloading wouldn't work--changes in the source code on the host don't propagate until the image is rebuilt. Our docs do cover how to [enable hot-reloading](https://code.visualstudio.com/docs/containers/debug-python#_how-to-enable-hot-reloading-in-django-or-flask-apps) for Django and Flask, but the user has to remember to uncomment the line in the Dockerfile before they can build an image that's useful for anything other than debugging (i.e., deploying to production).